src/app/server/create-server.js

```diff
@@ -157,7 +157,7 @@ function createServer (app, BrowserWindow, ipcMain, opts) {
       }
 
       pending++
-      comp._module.parse(body, compOpts, sendToRenderer)
+      comp._module.parse(body, compOpts, sendToRenderer, req)
```

Comment:
Hmm. I would prefer leaving `sendToRenderer` as the last argument, as callbacks `sendToRenderer` as usually passed last. 

I think having 

```js
 comp._module.parse(body, req, compOpts, sendToRenderer)
```

would be best ... and don't forget to update:

https://github.com/plotly/orca/blob/master/CONTRIBUTING.md#nomenclature-for-ipc-callbacks

accordingly.

Reply:
Done in c1e0ab5